packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -33,6 +33,7 @@ import {
     Statistics,
     purgeDefaultStorages,
     validators,
+    RetryRequest,
```

Comment:
I haven't noticed before, but shouldn't this be named `RetryableError` or `RetryRequestError` or something?